sequana/scripts/main/enrichment_kegg.py

```diff
@@ -149,6 +150,24 @@ def enrichment_kegg(**kwargs):
 
     output_directory = kwargs["output_directory"]
     for compa, gene_dict in gene_lists.items():
+
+        if keggname.startswith('vc'):
+            logger.warning("Vibrio Cholera annotation old_locus_tag processed to be compatible with KEGG/Sequana")
```

Comment:
dsl ... Je ne comprend pas très bien le warning ...

Reply:
All vibrio cholera in kegg starts with the vc letters. This is hackish but seems to work for now.